src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.cpp

```diff
@@ -89,6 +89,7 @@ ShenandoahHeapRegion::ShenandoahHeapRegion(HeapWord* start, size_t index, bool c
     SpaceMangler::mangle_region(MemRegion(_bottom, _end));
   }
   _recycling.unset();
+  _need_bitmap_reset = true;
```

Comment:
Move to initializers? Why does it start with `true`? A new region would have a clear bitmap, right?